python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -260,6 +260,8 @@ def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
         desc = f"- {self.op.name}: {active} active, {queued} queued"
+        if self.op._in_task_submission_backpressure:
+            desc += ", BACKPRESSURED"
```

Comment:
to indicate that being backpressured is related to a high number of queued blocks, maybe we can group the text as follows. thoughts?
```suggestion
            desc += " (BACKPRESSURED)"
```

Reply:
I like that, will leave open to discussion but if no one objects will go forward with that.